docs/index.md

```diff
@@ -1,23 +1,40 @@
 <frontmatter>
-  footer: userGuideFooter.md
-  siteNav: userGuideSections.md
+  title: "Generate Instructional Websites from Markdown Text"
+  footer: footer.md
 </frontmatter>
 
 <include src="./common/header.md" />
 
 <div class="website-content">
 
-# **MarkBind**
+<h1 class="display-3"><md>**MarkBind**</md></h1>
 
-#### Generate dynamic websites from markdown text.
+<span class="lead">
 
-MarkBind is a website generator that can generate a website from markdown documents. While there are other markdown-to-html website generators around, **MarkBind can generate more dynamic websites**, as opposed to one-size-fits-all static content. MarkBind is particularly suitable to create content-heavy websites %%e.g., eLearning websites, online instruction manuals, project documentation etc.%% that aim for _<tooltip content="i.e., the reader can go deeper or get more content as desired">self-directed consumption</tooltip>_.   
+<big>**Generate dynamic websites from Markdown text.**</big>
 
-## <span class="glyphicon glyphicon-th-list" aria-hidden="true"></span> Features
+MarkBind can use Markdown documents to generate a <tooltip content="as opposed to _one-size-fits-all_ static content">_more dynamic_</tooltip> websites that facilitates _<tooltip content="i.e., the reader can chart their own path through the content and consume more/less content as desired">self-directed consumption</tooltip>_. MarkBind is ideal for creating content-heavy instructional websites %%e.g., eLearning websites, online instruction manuals, project documentation etc.%%.
```

Comment:
Add a `<br>` before 'e.g., eLearning .....' so the greyed text is all on 1 line.